### PR TITLE
Export Confluent Utility

### PR DESCRIPTION
**Pull request by SafarMirek**
This PR adds a new utility for exporting confluent schema registry into zip file compatible with Apicurio Registry import admin interface.

### CODE REVIEW

File: utils/exportConfluent/pom.xml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        <dependency>
+            <groupId>io.apicurio</groupId>
+            <artifactId>apicurio-registry-utils-import-export</artifactId>
+            <version>2.0.1.Final</version>
+        </dependency>
+
+        <dependency>
+            <groupId>io.apicurio</groupId>
+            <artifactId>apicurio-registry-common</artifactId>
+            <version>1.3.2.Final</version><!--$NO-MVN-MAN-VER$-->
```
Comment (carlesarnal) [inline utils/exportConfluent/pom.xml:45]: Why 1.3.2.Final?

File: utils/exportConfluent/pom.xml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        </dependency>
+
+        <dependency>
+            <groupId>io.apicurio</groupId>
+            <artifactId>apicurio-registry-app</artifactId>
+            <version>1.3.2.Final</version><!--$NO-MVN-MAN-VER$-->
```
Comment (carlesarnal) [inline utils/exportConfluent/pom.xml:51]: Same here

File: utils/exportConfluent/src/main/java/io/apicurio/registry/utils/export/Export.java [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+    }
+
+    protected ContentHandle canonicalizeContent(ArtifactType artifactType, ContentHandle content) {
+        try {
+            ArtifactTypeUtilProvider provider = factory.getArtifactTypeProvider(artifactType);
+            ContentCanonicalizer canonicalizer = provider.getContentCanonicalizer();
+            ContentHandle canonicalContent = canonicalizer.canonicalize(content);
+            return canonicalContent;
+        } catch (Exception e) {
+            e.printStackTrace();
```
Comment (carlesarnal) [inline utils/exportConfluent/src/main/java/io/apicurio/registry/utils/export/Export.java:234]: Change this, please

**Review by carlesarnal — APPROVED** [review]
LGTM. Nice job.

**Comment by carlesarnal** [comment]
@SafarMirek I have one doubt about this. What happens if a schema in confluent has references? I don't think we're dealing with that here in any way. Any ideas on that? I'm afraid we'll only be able to deal with that once we have that support in Apicurio Registry.

**Comment by EricWittmann** [comment]
Hey @SafarMirek I'm closing this PR as part of a cleanup we're doing of our list of PRs.  Let's move the work to a feature branch at some point!

**Pull request closed without merging**